recipes/jb-everything/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "jb-everything" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jb-everything-{{ version }}.tar.gz
+  sha256: efe53c547aae8794372817182db86822e039349ad6099ce4c9f0cd2e37f1c21a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dabl ~=0.1.9
+    - dask[complete] ~=2.30.0  # to be upstreamed
+    - eli5 ~=0.10.1
+    - jax ~=0.2.7
+    - lightgbm ~=3.1.0
+    - matplotlib-base ~=3.3.3
+    - missingno ~=0.4.2
+    - numpy ~=1.19.4
+    - pandas ~=1.1.4
+    - python >=3.5  # to be upstreamed
```

Comment:
If this is not a `noarch: python` recipe, then the restrictions on the python version should be up in the `skip` statement above.